Options.lua

```diff
@@ -2490,6 +2500,13 @@ local function CreateArenaWidgetOptions()
                 type = "toggle",
                 arg = {"arenaWidget", "HideName"},
               },
+              UseFrameSort = {
+                name = L["Use FrameSort"],
+                order = 30,
+                type = "toggle",
+                arg = {"arenaWidget", "UseFrameSort"},
+                hidden = function() return GetAddOnEnableState("FrameSort", UnitName("player")) == 0 end,
```

Comment:
What do you think of not hiding the option here, but just disabling it? With hiding, users won't see that such an option is actually available when the install FrameSort?

Reply:
Sounds good, got an example of how to disable?

Reply:
Instead of "hidden = ..." use "disabled = ...". The function should work as is, I think.

Reply:
That worked, done.